src/configurations/destinations/amazon_audience/db-config.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "AMAZON_AUDIENCE",
+  "displayName": "Amazon Audience",
+  "config": {
+    "auth": { "type": "OAuth", "role": "amazon_audience", "rudderScopes": ["delivery"] },
+    "transformAtV1": "router",
+    "excludeKeys": [],
+    "supportedSourceTypes": ["cloud", "warehouse"],
```

Comment:
we can remove `cloud` source support for audience destinations

Reply:
removed